examples/quilljs/index.js

```diff
@@ -1,69 +1,195 @@
-if(Quill == null){
-  alert("You must download quilljs! It needs to be in the same directory as y-richtext!");
+'use strict';
+
+if (window.Quill === null) {
+  window.alert("You must download quilljs! It needs to be in the same directory as y-richtext!");
 }
 
+var colors = ['rgb(0, 0, 0)', 'rgb(230, 0, 0)', 'rgb(255, 153, 0)',
+  'rgb(255, 255, 0)', 'rgb(0, 138, 0)', 'rgb(0, 102, 204)',
+  'rgb(153, 51, 255)', 'rgb(255, 255, 255)', 'rgb(250, 204, 204)',
+  'rgb(255, 235, 204)', 'rgb(255, 255, 204)', 'rgb(204, 232, 204)',
+  'rgb(204, 224, 245)', 'rgb(235, 214, 255)', 'rgb(187, 187, 187)',
+  'rgb(240, 102, 102)', 'rgb(255, 194, 102)', 'rgb(255, 255, 102)',
+  'rgb(102, 185, 102)', 'rgb(102, 163, 224)', 'rgb(194, 133, 255)',
+  'rgb(136, 136, 136)', 'rgb(161, 0, 0)', 'rgb(178, 107, 0)',
+  'rgb(178, 178, 0)', 'rgb(0, 97, 0)', 'rgb(0, 71, 178)',
+  'rgb(107, 36, 178)', 'rgb(68, 68, 68)', 'rgb(92, 0, 0)',
+  'rgb(102, 61, 0)', 'rgb(102, 102, 0)', 'rgb(0, 55, 0)',
+  'rgb(0, 41, 102)', 'rgb(61, 20, 102)'];
+
+// Fill the palette
+$('.ql-color, .ql-background').each(function () {
+  var i, generate;
+  generate = function (what) {
+    return '<option value="' + what + '" label="' + what + '"></option>';
+  };
+  for (i = 0; i < colors.length; i++) {
+    $(this).append(generate(colors[i]));
+  }
+});
+$('.ql-color option[label="rgb(255,255,255)"]').select();
+
 var quill = new Quill('#editor', {
-    modules: {
-        'multi-cursor': true,
-        'link-tooltip': true,
-        'image-tooltip': true
-    },
-    theme: 'snow'
+  modules: {
+    'link-tooltip': true,
+    'image-tooltip': true
+  },
+  theme: 'snow'
 });
 quill.addModule('toolbar', { container: '#toolbar' });
-window.connector = new Y.WebRTC('thisisMYroom2');
+quill.addModule('multi-cursor', { timeout: 2500});
+window.connector = new Y.WebRTC('thisIsMyRoom2');
+//{url: 'http://localhost:8888'});
 
 // connector.debug = true;
-window.y = new Y(connector);
-
-// will perform a check if quill & y-richtext are equal.
-// We do weak comparison only, therefore (1 == "1").
-checkConsistency = function(){
-  deltas = editor.getDelta()
-  quill_deltas = quill.getContents().ops
-  for(d in deltas){
-    delta = deltas[d]
-    for(name in delta){
-      value = delta[name]
-      quill_value = quill_deltas[d][name]
-      if(value.constructor === Object){
-        for(n in value){
-          if(value[n] != quill_value[n]){
-            return false
+window.y = new Y(window.connector);
+
+var checkConsistency = function () {
+  var deltas = window.editor.getDelta(),
+    quill_deltas = quill.getContents().ops,
+    d,
+    delta, name, quill_value, value, n;
+  for (d in deltas) {
+    delta = deltas[d];
+    for (name in delta) {
+      value = delta[name];
+      quill_value = quill_deltas[d][name];
+      if (value.constructor === Object) {
+        for (n in value) {
+          if (value[n] !== quill_value[n]) {
```

Comment:
It is supposed to be `!=` not `!==`. As I wrote in the description you deleted .. 
I need this to check consistency. Otherwise the test will fail if you insert an image. Since I wrote that so explicitely, why did you remove it?